lib/util.js

```diff
@@ -374,3 +374,63 @@ exports.parseAbbreviatedNumber = string => {
   }
   return null;
 };
+
+/**
+ * Match begin and end braces of input JSON, return only json
+ *
+ * @param {String} mixedJson
+ * @return {String}
+*/
+
+exports.cutAfterJSON = mixedJson => {
+  let open, close;
+  if (mixedJson[0] === '[') {
+    open = '[';
+    close = ']';
+  } else if (mixedJson[0] === '{') {
+    open = '{';
+    close = '}';
+  }
+
+  if (!open) {
+    throw new Error(`Can't cut unsupported JSON (need to begin with [ or { ) but got: ${mixedJson[0]}`);
+  }
+
+  // States if the loop is currently in a string
+  let isString = false;
+
+  // Current open brackets to be closed
+  let counter = 0;
+
+  let i;
+  for (i = 0; i < mixedJson.length; i++) {
+    // Toggle the isString boolean when leaving/entering string
+    if (mixedJson[i] === '"' && mixedJson[i - 1] !== '\\') {
+      isString = !isString;
+      continue;
+    }
+    if (isString) continue;
+
+    if (mixedJson[i - 1] === '\\') continue;
```

Comment:
codecov shows this line is never reached. and that makes sense, there is only escaping inside strings, and if we're inside a string, we `continue` in the line above